domains/spreadsheets600.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "Personal Portfolio Website",
+    "repo": "https://github.com/SpreadSheets600/SpreadSheets600",
+    "owner": {
+        "username": "SpreadSheets600",
+        "email": "sohammaity006@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://spreadsheets600.github.io/SpreadSheets600/"
```

CNAME shouldn't contain protocol or `/`
```suggestion
        "CNAME": "spreadsheets600.github.io"
```